scripts/createTypescriptProgram.js

```diff
@@ -0,0 +1,41 @@
+const tmp = require("tmp");
+const ts = require("typescript");
+const path = require("path");
+const fs = require("fs-extra");
+
+tmp.setGracefulCleanup(); // cleans tmp file on process exit
+exports.corvidDir = path.join(__dirname, "../");
+
+const prepareEmptyTypescriptProgram = configPath => {
+  const tmpDir = tmp.dirSync();
+
+  const tmpDirPath = tmpDir.name;
+
+  const tmpConfigPath = `${tmpDirPath}/tsconfig.json`;
+  const tmpConfigContent = `{"extends": "${path.resolve(
+    exports.corvidDir,
+    configPath
+  )}"}`;
+
+  fs.writeFileSync(tmpConfigPath, tmpConfigContent);
+  fs.writeFileSync(`${tmpDirPath}/empty.js`, "");
+
+  return tmpConfigPath;
+};
+
+exports.createTsProgram = tsConfigPath => {
```

Comment:
I suggest just making it the single export from this file, so you can require it directly (since it's a file containing a single function)